cpp/data/sql_updates/ub_tools.28

```diff
@@ -0,0 +1 @@
+ALTER TABLE zeder_journals CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_bin;
```

Comment:
Should there be a corresponding change in ub_tools.sql for the collation?

Reply:
Apparently we inherit the character set and collation method from the ub_tools database and therefore there is no problem in ub_tools.sql.